chroma_core/models/repo.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019 DDN. All rights reserved.
+# Use of this source code is governed by a MIT-style
+# license that can be found in the LICENSE file.
+
+
+from django.db import models
+
+
+class Repo(models.Model):
+    repo_name = models.CharField(primary_key=True, max_length=50, help_text="Unicode string, repo name")
+    location = models.CharField(max_length=255, help_text="Unicode string, repo file location")
+
+    class Meta:
+        unique_together = ("repo_name",)
```

Comment:
Is this supposed to have only one value?

Reply:
yes.  `repo_name` is primary key, and that's how it seems to be written in other places.